selenium/che-selenium-test/src/test/resources/suites/CheOneThreadTestsSuite.xml

```diff
@@ -58,12 +58,9 @@
             <class name="org.eclipse.che.selenium.dashboard.organization.AddWorkspaceToOrganizationTest"/>
             <class name="org.eclipse.che.selenium.dashboard.organization.ShareWorkspaceTest"/>
             <class name="org.eclipse.che.selenium.git.AuthorizeOnGithubFromDashboardTest"/>
-            <class name="org.eclipse.che.selenium.git.AuthorizeOnGithubFromImportTest"/>
+            <class name="org.eclipse.che.selenium.git.ImportWizardFormTest"/>
```

Comment:
`ImportWizardFormTest` can be moved into the CheSuite.xml so as it's ready for multi-thread execution.